x-pack/test_serverless/functional/test_suites/observability/dataset_quality/home.ts

```diff
@@ -16,17 +17,42 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   ]);
 
   const testSubjects = getService('testSubjects');
+  const synthtrace = getService('svlLogsSynthtraceClient');
+  const to = '2024-01-01T12:00:00.000Z';
 
   describe('Dataset quality home', () => {
     before(async () => {
       await PageObjects.svlCommonPage.loginWithRole('admin');
     });
 
-    it('dataset quality table exists', async () => {
-      await PageObjects.datasetQuality.navigateTo();
-      await testSubjects.existOrFail(
-        PageObjects.datasetQuality.testSubjectSelectors.datasetQualityTable
-      );
+    describe('with no datasets available', () => {
+      before(async () => {
+        await PageObjects.datasetQuality.navigateTo();
+      });
+
+      it('shows the empty state', async () => {
+        await testSubjects.existOrFail(
+          PageObjects.datasetQuality.testSubjectSelectors.datasetQualityNoDataEmptyState
+        );
+      });
+    });
+
+    describe('with datasets available', () => {
+      before(async () => {
+        await synthtrace.index(getInitialTestLogs({ to, count: 1 }));
+        await PageObjects.datasetQuality.navigateTo();
+      });
+
+      after(async () => {
+        await synthtrace.clean();
+      });
+
+      it('dataset quality table exists', async () => {
+        await PageObjects.datasetQuality.navigateTo();
+        await testSubjects.existOrFail(
+          PageObjects.datasetQuality.testSubjectSelectors.datasetQualityTable
+        );
+      });
     });
```

Comment:
nit: As this test file is super small and only offer 2 tests, we can make it simple. Haven't tested it, but something like this

```
  describe('Dataset quality home', () => {
    before(async () => {
      await PageObjects.svlCommonPage.loginWithRole('admin');
      await PageObjects.datasetQuality.navigateTo();
    });

    it('shows the empty state when no dataset is available', async () => {
      await testSubjects.existOrFail(
        PageObjects.datasetQuality.testSubjectSelectors.datasetQualityNoDataEmptyState
      );
    });

    it('shows the dataset quality table when available', async () => {
      await synthtrace.index(getInitialTestLogs({ to, count: 1 }));
      await PageObjects.datasetQuality.refreshTable();

      await testSubjects.existOrFail(
        PageObjects.datasetQuality.testSubjectSelectors.datasetQualityTable
      );

      await synthtrace.clean();
    });
  });
```

Reply:
I was thinking that the sub-suite `with datasets available` will have more tests soon as we are moving the Dataset Quality under Stack Management, so this offers a bit of structure for that.